build/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+""" Builds all Fuchsia artifacts vended by Flutter.
+"""
+
+import argparse
+import subprocess
+import os
+import errno
+import sys
+import shutil
+
+src_root_dir = os.path.abspath(os.path.join(os.path.realpath(__file__), '..', '..', '..'))
```

Comment:
Global variables should have a "_" prepended: http://google.github.io/styleguide/pyguide.html#25-global-variables